.github/workflows/luajit-integration.yml

```diff
@@ -0,0 +1,79 @@
+name: LuaJIT integration testing
```

Comment:
I think it will be useful to add some comments about where this workflow is used to the beginning of the file.